templates/blog/macros.html

```diff
@@ -22,7 +22,7 @@
         <img src="{{ article.image | default('/static/uploads/articles/default.png', true) }}" width=100%>
       </figure>
       {{ article.content|striptags|truncate(200) }}
-    <a href={{ url }} lang={{ article.language }} class='read action-button'>Read more</a>
+    <a href={{ url_for_article(article) }} lang={{ article.language }} class='read action-button'>Read more</a>
```

Comment:
double quotes for html when quotes are required

Reply:
so href requires quotes I think it's the same for `src` in `img` (isn't?)

Reply:
yes, the rule is simple:

1. don't use quotes unless required
2. when required, use double quotes

Reply:
how you know if it's require ? like I know for class I need it when we have a space inside ... same for src.
There are other attribute ?

Reply:
You know the rule, same for any attribute: if there's a caracter that would break the html in the value of the attribute, you need quotes. Such as a space, a `=`, a `<`, ` >, ` /, …